### DISCUSSION TITLE
How to "customize" the federated or public timeline

### DISCUSSION THREAD

**Discussion by EismerSt** [mastodon/mastodon#17332] [Q&A] [answered]
Greetings to all!

I am absolutly new to Mastodon and want to run my own instance for local topics. Is there a way to customize federated (or public?) timeline, so that visitors or users see there public posts from chosen instances with similar topics?

I test-followed a user. Now his content is shown in the federated timeline, no matter what I do against this as admin. Further more, I added lots of relais and most of you can image what's going on on my federated timeline. How can I repair that damage?

Kind regards

Eismer

**Comment by ClearlyClaire** [comment, marked as answer]
As a moderator, you can “limit”/“silence” accounts or whole instances, which will prevent their posts from appearing in the federated or public timeline (but this will also prevent them from generating notifications to user they do not follow; those two things are unfortunately tied together at the moment).

There is no other way to customize the local or federated public timelines, they are meant to be a “firehose” view of the public posts known to your instance.

I guess you could also work with similar-minded instance admins to set up relays on your topics, but there is currently no way to follow an entire instance.

**Reply by EismerSt** [reply]
Thank you for your very quick answer. You helped me a lot. Now I'm thinking about running my own relay.

Have a nice weekende!

Eismer
